Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml

```diff
@@ -2027,6 +2045,683 @@ script:
+    description: Returns the trusted registries, repositories, and images. Maps to the image table in Defend > Compliance > Trusted Images in the Console UI.
+    outputs:
+    - contextPath: PrismaCloudCompute.TrustedImage.policy.enabled
+      description: Whether the trusted image policy is enabled.
+      type: Boolean
+    - contextPath: PrismaCloudCompute.TrustedImage.policy._id
+      description: The ID of the trusted image policy.
+      type: String
+    - contextPath: PrismaCloudCompute.TrustedImage.policy.rules.name
+      description: The name of the trusted image rule.
+      type: String
+    - contextPath: PrismaCloudCompute.TrustedImage.policy.rules.allowedGroups
+      description: The allowed groups for the trusted image rule.
+      type: Unknown
+    - contextPath: PrismaCloudCompute.TrustedImage.policy.rules.effect
+      description: The effect of the trusted image rule.
+      type: String
+    - contextPath: PrismaCloudCompute.TrustedImage.policy.rules.modified
+      description: The last modified timestamp for the trusted image rule.
+      type: Date
+    - contextPath: PrismaCloudCompute.TrustedImage.policy.rules.previousName
+      description: The previous name of the trusted image rule.
+      type: String
+    - contextPath: PrismaCloudCompute.TrustedImage.policy.rules.owner
+      description: The owner of the trusted image rule.
+      type: String
+    - contextPath: PrismaCloudCompute.TrustedImage.policy.rules.disabled
+      description: Whether the trusted image rule is disabled.
+      type: Boolean
+    - contextPath: PrismaCloudCompute.TrustedImage.policy.rules.collections
+      description: The collections for the trusted image rule.
+      type: Unknown
+    - contextPath: PrismaCloudCompute.TrustedImage.groups.modified
+      description: The last modified timestamp for the trusted image group.
+      type: Date
+    - contextPath: PrismaCloudCompute.TrustedImage.groups.owner
+      description: The owner of the trusted image group.
+      type: String
+    - contextPath: PrismaCloudCompute.TrustedImage.groups.name
+      description: The name of the trusted image group.
+      type: String
+    - contextPath: PrismaCloudCompute.TrustedImage.groups.previousName
+      description: The previous name of the trusted image group.
+      type: String
+    - contextPath: PrismaCloudCompute.TrustedImage.groups._id
+      description: The ID of the trusted image group.
+      type: String
+    - contextPath: PrismaCloudCompute.TrustedImage.groups.images
+      description: The images in the trusted image group.
+      type: Unknown
+  - name: prisma-cloud-compute-trusted-images-update
+    description: Updates a trusted image to the system. Specify trusted images using either the image name or layers properties. This is a potentially harmful command which overwrites the existing list, so use with caution. We recommend that is only be used in a playbook by a script that uses it.
```

Comment:
@ShirleyDenkberg please review this description.

Reply:
I asked Shirley about it on Slack, she told me what to add here.